.github/workflows/chromatic-build.yml

```diff
@@ -41,13 +41,20 @@ jobs:
       with:
         node-version: ${{ matrix.node-version }}
 
+    # Build the project
+    - name: Build
+      run: yarn build
+    - name: Build Storybook
+      run: yarn build:storybook
+
     # (if) Run this step on non-dependabot PRs.
     - name: Publish to Chromatic for visual testing (review PRs only)
       if: ${{ inputs.target == 'review' }}
       uses: chromaui/action@v1
       with:
         token: ${{ secrets.GITHUB_TOKEN }}
         projectToken: ${{ secrets.projectToken }}
+        storybookBuildDir: storybook-static
```

Comment:
**note:** This was required by Chromatic due to some internal changes in SB7 (+ the pre-step for building modules + building storybook)